### PR TITLE
Creation of Recruitment page for Fall 2020

### PR DESCRIPTION
**Pull request by Littlefieldm**
Making a Recruitment page with online links for Fall 2020 Rush week due to COVID related issues.

### CODE REVIEW

File: static/recruitment/index.html [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+<!-- Main Content -->
+<div class="content-wrapper">
+
+    <!-- RUSH Event 1: Taste of the Quad Fall 2020 -->
+    <div class="content">
+        <h2 class="content-head is-center">Taste of the Quad!</h2>
+        <h4 class="event-content-subhead is-center">Saturday August 22, 2020</h4>
+        <h4 class="event-content-subhead is-center">3:00 PM</h4>
+        <h4 class="event-content-subhead is-center"><a href="">ONLINE</a></h4>
```
Comment (gabrielkrell) [inline static/recruitment/index.html:70]: placeholder?

File: static/recruitment/index.html [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+        </div>
+    </div>
+
+    <!-- RUSH Event 2: Q&A FALL2020 -->
+    <div class="ribbon pure-g l-box-lrg">
+        <div class="pure-u-1">
+          <h2 class="content-head-ribbon is-center">Q and A Social Media Giveaway!</h2>
+          <h4 class="content-head-ribbon is-center">Sunday August 23, 2020</h4>
+          <h4 class="content-head-ribbon is-center">5:00 PM</h4>
+          <h4 class="content-head-ribbon is-center"><a href="https://docs.google.com/document/d/1in4xA7A-eHMzziszb6KpVL21hsdfnZGTpzb9y-UVB8Q/edit?usp=sharing">ONLINE</a></h4>
```
Comment (gabrielkrell) [inline static/recruitment/index.html:84]: If you don't want the docs menu bar on desktop, can do `/preview` in the link

Comment (gabrielkrell) [inline static/recruitment/index.html:84]: Also, default hyperlink blue and purple are really hard to read on top of navy background

File: static/recruitment/index.html [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+    <link rel="stylesheet" href="https://netdna.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.css">
+
+    <link href='https://fonts.googleapis.com/css?family=Roboto+Slab' rel='stylesheet' type='text/css'>
+
+    <link href='https://fonts.googleapis.com/css?family=Oswald' rel='stylesheet' type='text/css'>
+
+    <link href='https://fonts.googleapis.com/css?family=Roboto:400italic,400,700' rel='stylesheet' type='text/css'>
+
+    <!-- Import css -->
+    <link rel="stylesheet" href="../css/layouts/aepks.css">
```
Comment (gabrielkrell) [inline static/recruitment/index.html:35]: Might want to add a diff background image. Maybe Haney has something, or could grab an old rush photo from the drive

File: static/data/brothers_2020-08-17.json [hunk 4]
```diff
@@ -0,0 +1,67 @@
+{
```
Comment (gabrielkrell) [inline static/data/brothers_2020-08-17.json:1]: Please remove the old json when you update - we used to have a whole mess of different JSONs with different naming schemes that accumulated. better to just keep it in git history.

File: static/data/brothers_2020-08-17.json [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+  {"lastname": "Grudowski", "name": "Noah Grudowski", "positions": [], "classOf": "AE 2021", "visible": true, "hasImg": true},
+  {"lastname": "Gutierrez", "name": "Rey Gutierrez", "positions": [], "classOf": "AE 2023", "visible": true, "hasImg": false},
+  {"lastname": "Haney", "name": "Zachary Haney", "positions": ["Chi"], "classOf": "AE 2021", "visible": true, "hasImg": true},
+  {"lastname": "Hawn", "name": "Ethan Hawn", "positions": ["Theta 2"], "classOf": "AE 2023", "visible": true, "hasImg": false},
+  {"lastname": "Ishmam", "name": "Alif Ishmam", "positions": [], "classOf": "AE 2023", "visible": true, "hasImg": false},
+  {"lastname": "Kanofsky", "name": "Cameron Kanofsky", "positions": [], "classOf": "AE 2023", "visible": true, "hasImg": false},
+  {"lastname": "Keats", "name": "Daniel Keats", "positions": ["Iota"], "classOf": "AE 2021", "visible": true, "hasImg": true},
+  {"lastname": "Klompenhower", "name": "Daniel Klompenhower", "positions": [], "classOf": "AE 2022", "visible": true, "hasImg": true},
+  {"lastname": "Krell", "name": "Gabriel Krell", "positions": [], "classOf": "AE 2022", "visible": true, "hasImg": true},
```
Comment (gabrielkrell) [inline static/data/brothers_2020-08-17.json:29]: take out alumni?

**Review by gabrielkrell — COMMENTED** [review]
Surprised I still get notifications. Here are some comments. I will let Justin or a past epsilon merge

**Comment by gabrielkrell** [comment]
also good luck not getting covid during rush

**Comment by aurangzebrazzaq** [comment]
Deploy preview looks good

**Pull request merged**